examples/landmark_detection/example_scan.py

```diff
@@ -0,0 +1,218 @@
+# %%
+
+import numpy as np
+import pandas as pd
+
+
+from pathlib import Path
+
+from giskard_vision.scanner.scanner_vision import Scanner
+from giskard_vision.landmark_detection.models.wrappers import OpenCVWrapper
+from giskard_vision.landmark_detection.dataloaders.loaders import DataLoader300W
+
+from face_alignment import FaceAlignment, LandmarksType
+
+from giskard_vision.landmark_detection.dataloaders.wrappers import (
+    CroppedDataLoader,
+    ResizedDataLoader,
+    ColoredDataLoader,
+    BlurredDataLoader,
+    FilteredDataLoader,
+    HeadPoseDataLoader,
+    EthnicityDataLoader,
+    CachedDataLoader,
+)
+from giskard_vision.landmark_detection.tests.performance import NMEMean
+from giskard_vision.landmark_detection.tests.base import Test, TestDiff
+from giskard_vision.landmark_detection.marks.facial_parts import FacialParts
+from giskard_vision.scanner.scanner_vision import Scanner
+
+from giskard.scanner.issues import Issue, IssueLevel, IssueGroup
+
+# %%
+
+
+class FacialPartsDetector:
+    def __init__(self):
+        pass
+
+    def run(self, model, dataset):
+        facial_parts = [FacialParts.LEFT_HALF.value, FacialParts.RIGHT_HALF.value]
+
+        issues = []
+
+        for fp in facial_parts:
+            dl = CroppedDataLoader(dataset, part=fp)
+            result = TestDiff(metric=NMEMean, threshold=1).run(
+                model=model,
+                dataloader=dl,
+                dataloader_ref=dataset,
+                facial_part=fp,
+            )
+            result = result.to_dict()
```

Comment:
no need to convert to dict. You can retrieve all the info you need from `result` (which should have additional attributes than the one taken in `to_dict()`)